otsdaq/FECore/FEVInterface.cc

```diff
@@ -626,11 +626,16 @@ const std::string& FEVInterface::getFEMacroConstArgument(frontEndMacroConstArgs_
 // getFEMacroConstArgumentValue
 //	helper function for getting the copy of the value of an argument
 template<>
-std::string ots::getFEMacroConstArgumentValue<std::string>(FEVInterface::frontEndMacroConstArgs_t& args, const std::string& argName)
+std::string ots::getFEMacroConstArgumentValue<std::string>(FEVInterface::frontEndMacroConstArgs_t& args, 
+														   const std::string& argName,
+														   const std::string& defaultValue)
 {
-	return FEVInterface::getFEMacroConstArgument(args, argName);
+	const std::string& data = FEVInterface::getFEMacroConstArgument(args, argName);
+	if (data.compare("") == 0)
```

Comment:
Probably easier to understand as `if(data == "")`

Reply:
Thanks for the feedback, I change it and I also change few others stuff discussed with Ryan.